addons/web/static/src/views/helpers/view_compiler.js

```diff
@@ -766,6 +780,26 @@ export class ViewCompiler {
             }
         }
 
+        if (pageAnchors.length) {
+            // If anchors from the page are targetting an element
+            // present in the notebook, it must be aware of the
+            // page that contains the corresponding element
+            for (const anchor of pageAnchors) {
+                let pageId = 1;
+                for (const child of el.children) {
+                    if (child.querySelector(anchor)) {
```

Comment:
maybe I'm missing something - but this seems to have broken any form with a notebook?